src/Models/RidersModel.php

```diff
@@ -117,17 +167,62 @@ public function addRider(
                 'name' => $name,
                 'image' => $image,
                 'dob' => $dob,
-                'giroGcWins' => $giroGcWins,
-                'tourGcWins' => $tourGcWins,
-                'vueltaGcWins' => $vueltaGcWins,
-                'giroStageWins' => $giroStageWins,
-                'tourStageWins' => $tourStageWins,
-                'vueltaStageWins' => $vueltaStageWins
+                'giroGc' => $giroGc,
+                'tourGc' => $tourGc,
+                'vueltaGc' => $vueltaGc,
+                'giroStages' => $giroStages,
+                'tourStages' => $tourStages,
+                'vueltaStages' => $vueltaStages
+            ]
+        );
+    }
+
+    public function editRider(
+        int $id,
+        string $name,
+        string $image,
+        int $teamId,
+        int $nationId,
+        string $dob,
+        ?int $giroGc,
+        ?int $tourGc,
+        ?int $vueltaGc,
+        ?int $giroStages,
+        ?int $tourStages,
+        ?int $vueltaStages
+    ): bool {
+        $query = $this->db->prepare("
+            UPDATE `riders` SET
+            `name` = :name,
+            `image` = :image,
+            `team_id` = $teamId,
+            `nation_id` = $nationId,
```

Comment:
I think you should still bind these, I don't really see anyway anyone could actually use these two fields to perform an attack, but better to be safe than sorry!